data.sql

```diff
@@ -13,6 +13,8 @@ CREATE TABLE users
   first_name varchar(20) NOT NULL,
   last_name varchar(20) NOT NULL,
   location varchar(64) NOT NULL,
+  profile_icon varchar(256) NOT NULL,
```

Comment:
Why is this so massive?

Reply:
it really shouldn't be realisticaly.
Updated to be 16 characters, that should be 9 characters for upload/pp_ and 7 characters available for user Id's.